typescript.js

```diff
@@ -54,7 +54,7 @@ var extendedRules = Object.fromEntries(
 
 module.exports = {
 
-  extends: ['plugin:@typescript-eslint/recommended'],
+  extends: ['./module.js', 'plugin:import/typescript', 'plugin:@typescript-eslint/recommended'],
```

Comment:
The `typescript` preset now depends directly on the `module` preset, allowing `module` to apply its rules to TypeScript files as well. This is a breaking change.